### PR TITLE
Update events path

### PR DESCRIPTION
**Pull request by JesseGCarver**
Fixes for the event query as well as tests showing their correctness.

### CODE REVIEW

**Review by rhooper — COMMENTED** [review]
This will need pypackage.toml updating to 1.2.1.  Could you add that to the PR?

**Comment by mel-seto** [comment]
@JesseGCarver how do I test this change?

**Review by rhooper — COMMENTED** [review]
Looks okay to me now - I ran the tests and they looked good.  @mel-seto is having a second look, but we're good to go when they're done.

**Comment by mel-seto** [comment]
I'll get my environment working and then test today. I found the README.md and i think this is a good opportunity to get up-to-date on the python API client

File: config/config.ini [hunk 1]
```diff
@@ -3,10 +3,10 @@ token_url = /oauth2/token
 base_url = https://api.lexmachina.com
```
Comment (mel-seto) [inline config/config.ini:3]: As Rose noted in slack, this file should have been checked in

File: tests/_async/test_district_case.py [hunk 2]
```diff
@@ -17,6 +17,7 @@ class TestGetDistrictCase:
     async def test_get_district_case_integer_fuzz(self, cases):
 
         response = await self.client.get_district_cases(cases=cases)
+        print(response)
         assert response.get("detail") == "case_id not found"
```
Comment (mel-seto) [inline tests/_async/test_district_case.py:21]: Do we want this print statement in here?

Comment (JesseGCarver) [reply tests/_async/test_district_case.py]: fixed

**Review by mel-seto — CHANGES_REQUESTED** [review]
Test failure here for both the async and sync tests:
```
____________________________________________ test_query_district_case_events _____________________________________________

    @pytest.mark.asyncio
    async def test_query_district_case_events():
        query = DistrictCaseQueryRequest().include_event_types("Trial")
        response = await client.query_district_case(query)
>       assert len(response) == 5
E       TypeError: object of type 'NoneType' has no len()

test_query_district_case.py:30: TypeError
```

**Comment by mel-seto** [comment]
I pulled latest and still get the same error:

```
(apiclient2) melseto@Mel-Laptop _async % git pull
Already up to date.
(apiclient2) melseto@Mel-Laptop _async % git pull -f
Already up to date.
(apiclient2) melseto@Mel-Laptop _async % pytest test_query_district_case.py
================================================== test session starts ===================================================
platform darwin -- Python 3.11.6, pytest-8.0.1, pluggy-1.4.0
rootdir: /Users/melseto/Programming/python-lexmachina-sync-api-client/tests/_async
plugins: asyncio-0.23.5
asyncio: mode=Mode.STRICT
collected 3 items                                                                                                        

test_query_district_case.py ..F                                                                                    [100%]

======================================================== FAILURES ========================================================
____________________________________________ test_query_district_case_events _____________________________________________

    @pytest.mark.asyncio
    async def test_query_district_case_events():
        query = DistrictCaseQueryRequest().include_event_types("Filed")
        response = await client.query_district_case(query)
        print(response)
>       assert len(response) == 5
E       TypeError: object of type 'NoneType' has no len()

test_query_district_case.py:31: TypeError

```

**Comment by mel-seto** [comment]
@rhooper are you getting my error?

**Comment by rhooper** [comment]
@mel-seto are you testing against https://api.lexmachina.com?

**Comment by rhooper** [comment]
The None response is curious.

**Comment by JesseGCarver** [comment]
Suspect this is that your env has the pipy provided 1.2.1 version, need to do `pip install .` from the root (where `pyproject.toml` lives) to get this to use the version on disk when running tests.

All test pass on my local:
```
=========================================================== test session starts ============================================================
platform linux -- Python 3.11.6, pytest-8.0.1, pluggy-1.4.0
rootdir: /home/jesse/dev/python-lexmachina-sync-api-client
plugins: asyncio-0.23.5
asyncio: mode=Mode.STRICT
collected 98 items                                                                                                                         

tests/_async/test_attorneys.py .....                                                                                                 [  5%]
tests/_async/test_district_case.py ....                                                                                              [  9%]
tests/_async/test_get_judges.py ...............                                                                                      [ 24%]
tests/_async/test_get_lawfirms.py .....                                                                                              [ 29%]
tests/_async/test_get_parties.py .....                                                                                               [ 34%]
tests/_async/test_get_patents.py ....                                                                                                [ 38%]
tests/_async/test_health.py .                                                                                                        [ 39%]
tests/_async/test_list_case_resolutions.py .                                                                                         [ 40%]
tests/_async/test_list_case_tags.py .                                                                                                [ 41%]
tests/_async/test_list_case_types.py .                                                                                               [ 42%]
tests/_async/test_list_courts.py .                                                                                                   [ 43%]
tests/_async/test_list_damages.py .                                                                                                  [ 44%]
tests/_async/test_list_events.py .                                                                                                   [ 45%]
tests/_async/test_list_judgment_sources.py .                                                                                         [ 46%]
tests/_async/test_query_district_case.py ...                                                                                         [ 50%]
tests/_sync/test_attorneys.py .....                                                                                                  [ 55%]
tests/_sync/test_district_case.py ....                                                                                               [ 59%]
tests/_sync/test_get_judges.py ...............                                                                                       [ 74%]
tests/_sync/test_get_lawfirms.py .....                                                                                               [ 79%]
tests/_sync/test_get_parties.py .....                                                                                                [ 84%]
tests/_sync/test_get_patents.py ....                                                                                                 [ 88%]
tests/_sync/test_health.py .                                                                                                         [ 89%]
tests/_sync/test_list_case_resolutions.py .                                                                                          [ 90%]
tests/_sync/test_list_case_tags.py .                                                                                                 [ 91%]
tests/_sync/test_list_case_types.py .                                                                                                [ 92%]
tests/_sync/test_list_courts.py .                                                                                                    [ 93%]
tests/_sync/test_list_damages.py .                                                                                                   [ 94%]
tests/_sync/test_list_events.py .                                                                                                    [ 95%]
tests/_sync/test_list_judgment_sources.py .                                                                                          [ 96%]
tests/_sync/test_query_district_case.py ...                                                                                          [100%]

=========================================================== 98 passed in 54.17s ============================================================

```

**Pull request merged**